http/static.go

```diff
@@ -127,7 +131,19 @@ func getStaticHandlers(store *storage.Storage, server *settings.Server, assetsFs
 			return 0, nil
 		}
 
-		return handleWithStaticData(w, r, d, assetsFs, r.URL.Path, "application/javascript; charset=utf-8")
+		fileContents, err := fs.ReadFile(assetsFs, r.URL.Path+".gz")
+		if err != nil {
+			return http.StatusNotFound, err
+		}
+
+		w.Header().Set("Content-Encoding", "gzip")
```

Comment:
What if the browser doesn't support gzipped response?

Reply:
The content encoding and gzip was introduced long time ago in HTTP/1.1 (https://tools.ietf.org/html/rfc2616#section-3.5) and is probably supported by all browsers. (https://caniuse.com/sr_content-encoding-gzip).

We can use one of the following aproaches to make it optional:
- Have two variations of static files, increasing the shipped build size.
- Decode the gzip files on the fly, affecting performance.
- Encode the static files on the fly, affecting performance.

Reply:
I think that decoding gzip on the fly is a good fallback and wont affect performance since majority of the requests will be served as gzip. Decoding will only happen when the request does not support gzip.